src/main/java/org/humancellatlas/ingest/project/web/ProjectController.java

```diff
@@ -201,9 +201,12 @@
 
     @GetMapping(path = "projects/filter")
     public ResponseEntity<PagedResources<Resource<Project>>> filterProjects(
-            @ModelAttribute SearchFilter searchFilter,
+            @RequestParam(value = "search", required = false) String search,
```

Comment:
This is not a good practice, because we would need to modify the controller every time we add some search parameter. Let's take about it.

Reply:
@amnonkhen Do you mean to have only 1 search parameter would be better where we could somehow collect all the search criteria? Do you have an idea how that would look like?

Reply:
Yusra had fixed this. I thought I merged her changes but it seems otherwise. I will investigate

Reply:
please go back to using `@ModelAttribute`